swsnu/swpp18-team12
# Feature/account front

#### What does this PR do?
Implements account feature frontend
#### Description of Task to be completed?
- login page
  > /login
- signup page
  > /signup
- change password page
  > /account/change_password
- forgot password page
  > /account/forgot_password
- logout button
- basic nav header
- authguard
- alertService and alertComponent
#### How should this be manually tested?
1. Open backend server at proper branch
2. Open frontend server
3. Create new account with valid and invalid form at `/signup` (backend branch: master)
4. Try sign in at `/login`
5. Try change password with valid and invalid form at `/account/change_password`
6. Check every router links work well
7. Try logout at `/`
8. Try route to `/account/change_password` in logged out. You should be redirected to `/login`
9. Try reset password with valid and invalid email at `/account/forgot_password` (backend branch: feature/forgot/password)
